test/test_jra55.jl

```diff
@@ -96,11 +96,11 @@ using ClimaOcean.OceanSeaIceModels: PrescribedAtmosphere
 
         # Random regression test
         CUDA.@allowscalar begin
-            @test target_fts[1, 1, 1, 1] == 222.243136478611
+            @test Float32(target_fts[1, 1, 1, 1]) ≈ Float32(222.24313354492188)
 
             # Only include this if we are filling halo regions within
             # interpolate_field_time_series
-            @test target_fts[Nx + 1, 1, 1, 1] == 222.243136478611
+            @test Float32(target_fts[Nx + 1, 1, 1, 1]) ≈ Float32(222.24313354492188)
```

Comment:
isn't this a symptom that something has changed in the interpolation routine?

Reply:
maybe it's just that the indices are `Float32` now instead of `Float64`?

Reply:
yes thats right its a floating point precision thing

Reply:
the change is like 1e-5